lib/random.ts

```diff
@@ -58,6 +58,10 @@ export class Random {
     return this.choice(inclusiveRange(range));
   }
 
+  inInterval({ min, max }: NumericInterval): number {
```

Comment:
Nice, can you add a JSDoc-style comment here documenting it (here's an [example](https://github.com/toolness/mystic-symbolic/blob/3d8cca95165043510dd1049732c2540bdf6219df/lib/random.ts#L36-L39))?  Specifically it'd be helpful to say whether the endpoints of the range are inclusive/exclusive, since that can be hard to figure out just from the function signature.  Thanks!